master/buildbot/test/unit/test_db_logs.py

```diff
@@ -517,6 +517,12 @@ def test_addLogLines_huge_log_lots_line(self):
     def test_addLogLines_huge_log_lots_snowmans(self):
         return self.do_addLogLines_huge_log(NUM_CHUNKS=3000, chunk=u'\N{SNOWMAN}\n' * 50)
 
+    @defer.inlineCallbacks
+    def test_compressEmptyLog(self):
```

Comment:
This test tests the not existing log use case (which is nice to test too, but not coherent with the test name)

In order to test the EmptyLog use case, you need to insert the following in the db:

```python
        yield self.insertTestData(self.backgroundData + [
            fakedb.Log(id=201, stepid=101, name=u'stdio', slug=u'stdio',
                       complete=1, num_lines=0, type=u's'),
        ])
```